src/ProgressBar/ProgressBar.tsx

```diff
@@ -105,6 +119,23 @@ export default class ProgressBar extends React.PureComponent<Props> {
               }}
             >
               <div className={classnames(cssClass.BAR_BORDER, cssClass.borderSize(size))} />
+              {markNumbers &&
+                markNumbers.map((element, index) =>
+                  element < 100 ? (
```

Comment:
This looks like an eslint `no-confusing-arrow` violation https://eslint.org/docs/rules/no-confusing-arrow

I tend to agree with the sentiment of that lint rule, because the ternary operator makes it confusing. I recommend pulling the filtering logic up one level since we're just ignoring all of the elements that are too large by rendering a `null`.

```jsx
markNumbersInRange = (markNumbers || []).filter(mark => mark < 100)

{markNumbersInRange.map(...)}
```

Feel free to use whatever variable names you like, I just used those names for illustration purposes.

Reply:
`element < 100` is really just a safe gaurd against bad user input but I see your point that using filter will be better. 

about the eslint violation: Current code was in violation with eslint for that rule but the fix was in violation with the preetier package so `make format` was failing. Thats what the 3rd commit does. more context at https://clever.slack.com/archives/C0ERS5ABB/p1588281462092900